modules/sandboxed-containers-deploying-osc-with-other-ocp-components.adoc

```diff
@@ -0,0 +1,16 @@
+//Module included in the following assemblies:
+//
+// * sandboxed_containers/understanding_sandboxed_containers.adoc
+
+:_content-type: CONCEPT
+[id="sandboxed-containers-with-other-ocp-components_{context}"]
+
+= Using {sandboxed-containers-first} with {VirtProductName}
+//= Using {sandboxed-containers-first} with other {product-title} components
+
+You can use {sandboxed-containers-first} on clusters installed with {VirtProductName}.
```

Comment:
The {VirtProductName} attribute is not rendering. You need to include the virt attributes file in the assembly in order for it to render.